compiler/ExtractBase.ml

```diff
@@ -1970,20 +1984,20 @@ let ctx_add_generic_params (span : Meta.span) (current_def_name : Types.name)
 let ctx_add_decreases_proof (def : fun_decl) (ctx : extraction_ctx) :
     extraction_ctx =
   let name =
-    ctx_compute_decreases_proof_name def.span ctx def.def_id def.llbc_name
-      def.num_loops def.loop_id
+    ctx_compute_decreases_proof_name def.item_meta.span ctx def.def_id
```

Comment:
I think you should also use the `rename` attribute in `ctx_compute_decreases_proof_name`